hail/python/hail/docs/change_log.md

```diff
@@ -0,0 +1,35 @@
+# Change Log
+
+
+## 0.2.5 
+
+Released 2018-12-07
+
+### New features
+
+ - (hail #4845) The [or_error](https://hail.is/docs/0.2/functions/core.html#hail.expr.builders.CaseBuilder.or_error) method in `hl.case` and `hl.switch` statements now takes a string expression rather than a string literal, permitting nicer errors and assertions.
```

Comment:
GitHub formats PRs without a space: danking/hail#1 (my hail), danking/tea-script#1 (some other repo). I think we should stick to that convention.

Reply:
sure